src/app/diary-module/containers/sign-in/sign-in.component.scss

```diff
@@ -0,0 +1,91 @@
+#container {
+  width: 100vw;
+  height: 100vh;
+}
+
+#sign-in-card {
+  width: 48%;
+  height: 45%;
+  background-color: white;
+  border-radius: 10px;
+}
+
+.center {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+#logo {
+  width: 5vh;
+  padding: 1vh;
+}
+
+#logoSec {
+  margin: 1vh;
+  display: flex;
+  justify-content: flex-start;
+}
+
+#sign-in {
+  font-weight: lighter;
+  color: #039be5;
+  font-size: 35px;
+  text-align: center;
+}
+
+.grey-color {
+  color: gray;
+}
+
+#username-txt {
+  border-radius: 5px;
+  padding: 12px;
+  border: 1px solid silver;
+  font-size: medium;
+  width: 20vw;
+}
+
+input:focus-visible {
+  outline: 1px solid silver !important;
+}
+
+#random-btn {
+  margin-left: 2vw;
+  padding: 1.5vh;
+}
+
+#continue-btn {
+  font-size: 15px;
+  padding: 2vh;
```

Comment:
it's better stick to one unit type. (`px` or `rem`) not encourage to use `vh,vw` for padding and margins.